### PR TITLE
Produce import_metadata_mcf in executor

### PR DESCRIPTION
**Pull request by n-h-diaz**
Tested manually: gs://datcom-prod-imports/scripts/us_fed/treasury_constant_maturity_rates/USFed_ConstantMaturityRates_Test/import_metadata_mcf.mcf

Output will be in "import_metadata_mcf.mcf"

### CODE REVIEW

File: import-automation/executor/app/utils.py [hunk 1]
```diff
@@ -19,11 +19,14 @@
 import os
 import re
 import datetime
+from croniter import croniter
```
Comment (ajaits) [inline import-automation/executor/app/utils.py:22]: does this need 'croniter' in requirements.txt?

File: import-automation/executor/requirements.txt [hunk 2]
```diff
@@ -11,3 +11,5 @@ flask
 gunicorn
 pytz
 absl-py
+croniter
+deepdiff
```
Comment (ajaits) [inline import-automation/executor/requirements.txt:15]: Is this needed for any planned provenance mcf property:value?

File: import-automation/executor/app/executor/import_executor.py [hunk 3]
```diff
@@ -600,7 +627,7 @@ def _run_with_timeout(args: List[str],
         logging.exception(
             f'An unexpected exception was thrown: {e} when running {args}:'
             f' {message}')
-        return None
+        raise subprocess.TimeoutExpired(args, timeout)
```
Comment (ajaits) [inline import-automation/executor/app/executor/import_executor.py:630]: should it convert all exceptions to TimeoutExpired? 
Was this causing an error when testing manually?

**Review by ajaits — APPROVED** [review]
Thanks Natalie!

We'll remove the lastDataRefreshDate and nextDataRefreshDate from autorefresh import provenance once this is integrated.

**Comment by n-h-diaz** [comment]
> Thanks Natalie!
> 
> We'll remove the lastDataRefreshDate and nextDataRefreshDate from autorefresh import provenance once this is integrated.

Thanks for the review! It looks like I don't have merge authorization for this repo, so please feel free to merge whenever

**Pull request merged**